app/main.py

```diff
@@ -1,2 +1,75 @@
+from typing import Any, Hashable
+from copy import deepcopy
+
+
 class Dictionary:
-    pass
+    class Node:
+        def __init__(self, key: Hashable, value: Any, my_hash: int) -> None:
+            self.key = key
+            self.value = value
+            self.my_hash = my_hash
+
+        def __repr__(self) -> str:
+            return f"Node(key={self.key}, value={self.value})"
+
+    def __init__(self) -> None:
+        self.__capacity = 8
+        self.__load_factor = 2 / 3
+        self.__size = 0
+        self.__arr = [0 for _ in range(self.__capacity)]
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        if self.__ready_to_resize():
+            self.__resize()
+
+        index = self.__find_index(key)
+        if self.__arr[index] == 0:
+            self.__size += 1
+        self.__add_node(index, key, value)
```

Comment:
same here - it is a simple operation, you can set value here directly instead of creating function you never reuse